crates/node/src/shell/mod.rs

```diff
@@ -355,7 +355,14 @@ where
 
 /// Merkle tree storage key filter. Return `false` for keys that shouldn't be
 /// merklized.
-pub fn is_merklized_storage_key(key: &namada_sdk::storage::Key) -> bool {
+pub fn is_merklized_storage_key(_key: &namada_sdk::storage::Key) -> bool {
```

Comment:
We should just get rid of this function, honestly.

Reply:
Removed cd441a2